underfs/swift/src/main/java/alluxio/underfs/swift/SwiftInputStream.java

```diff
@@ -83,6 +93,9 @@ public int read(byte[] b, int offset, int length) throws IOException {
     int read = mStream.read(b, offset, length);
     if (read != -1) {
       mPos += read;
+      if (mPos % getBlockSize() == 0) {
+        closeStream();
+      }
```

Comment:
I would expect the Swift library to return 8, but it would be good to verify this.

@madanadit it would also be good to record the above mentioned invariant in the class documentation